### PR TITLE
node, p2p: move network config out of Server

### PR DESCRIPTION
**Pull request by fjl**
This silences a go vet message about copying p2p.Server in package node.

### CODE REVIEW

**Comment by robotally** [comment]
| Vote | Count | Reviewers |
| :---: | :---: | :---: |
| :+1: | 0 |  |
| :-1: | 0 |  |

_Updated: Wed May 18 09:33:10 UTC 2016_

**Comment by fjl** [comment]
@karalabe PTAL

File: p2p/server.go [hunk 1]
```diff
@@ -54,12 +54,7 @@ var errServerStopped = errors.New("server stopped")
 
 var srvjslog = logger.NewJsonLogger()
 
-// Server manages all peer connections.
-//
-// The fields of Server are used as configuration parameters.
-// You should set them before starting the Server. Fields may not be
-// modified while the server is running.
-type Server struct {
+type Config struct {
```
Comment (karalabe) [inline p2p/server.go]: Please add a doc :P

**Comment by karalabe** [comment]
Apart from the doc, LGTM :+1:

**Comment by fjl** [comment]
done

**Comment by karalabe** [comment]
Lol, nice :))

**Pull request merged**